Tests/ApolloCodegenTests/ApolloSchemaTests.swift

```diff
@@ -54,7 +57,8 @@ class ApolloSchemaTests: XCTestCase {
         "--endpoint=http://localhost:8080/graphql",
         "--key=\(apiKey)",
         "'\(expectedOutputURL.path)'",
-        "--header='\(header)'"
+        "--header='\(headers.first!)'",
+        "--header='\(headers.last!)'"
```

Comment:
I'd prefer these not have force-unwraps - maybe declare it as two vars and then shove those into the header array. However, this is something I can fix post-merge if you prefer.

Reply:
Too easy - have created two separate variables labelled `firstHeader` and `lastHeader` for clarity and to remove the force unwraps